rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_limits_urdf.py

```diff
@@ -33,12 +33,15 @@ def get_joint_limits(key='robot_description', use_smallest_joint_limits=True):
             if jtype == 'fixed':
                 continue
             name = child.getAttribute('name')
+            try:
+                limit = child.getElementsByTagName('limit')[0]
+            except:
+                continue
```

Comment:
We need the velocity limit to perform velocity scaling (bottom-most slider in plugin). On a `<limit>` [URDF tag](http://wiki.ros.org/urdf/XML/joint), effort and velocity attributes are required, position limits are optional.